pkg/morph/event/subnet/remove_node.go

```diff
@@ -0,0 +1,86 @@
+package subnet
+
+import (
+	"crypto/elliptic"
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/rpc/response/result/subscriptions"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neofs-node/pkg/morph/client"
+	"github.com/nspcc-dev/neofs-node/pkg/morph/event"
+	subnetid "github.com/nspcc-dev/neofs-sdk-go/subnet/id"
+)
+
+// RemoveNode structure of subnet.RemoveNode notification from morph chain.
+type RemoveNode struct {
+	subnetID *subnetid.ID
+	nodeKey  *keys.PublicKey
+
+	// txHash is used in notary environmental
+	// for calculating unique but same for
+	// all notification receivers values.
+	txHash util.Uint256
+}
+
+// MorphEvent implements Neo:Morph Event interface.
+func (RemoveNode) MorphEvent() {}
+
+// SubnetworkID returns a marshalled subnetID structure, defined in API.
+func (rn RemoveNode) SubnetworkID() *subnetid.ID { return rn.subnetID }
+
+// Node is public key of the nodeKey that is being deleted.
+func (rn RemoveNode) Node() *keys.PublicKey { return rn.nodeKey }
```

Comment:
We work only with `RemoveNode{}.Node().Bytes()`, so there is no need to interpret key bytes.

Reply:
changed